### PR TITLE
updating page with topic and author

### PR DESCRIPTION
**Pull request by DanielJPino-ux**
Adding topic and authors to page

This PR implements the following **changes:**

* adding "plain language" event topic tag
* adding Katherine Spivey and Leslie O'Flahavan as event authors

**URL / Link to page**

https://digital.gov/event/2019/06/12/how-develop-an-agency-brand-voice-that-connects-with-customers-online/

### CODE REVIEW

**Comment by DanielJPino-ux** [comment]
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/Event-Page-Update/event/2019/06/12/how-develop-an-agency-brand-voice-that-connects-with-customers-online/

File: content/events/2019/06/2019-06-12-how-develop-an-agency-brand-voice-that-connects-with-customers-online.md [hunk 1]
```diff
@@ -1,19 +1,42 @@
...
+# end date
+end_date: 2019-06-12 16:00:00 -0500
+
+# see all topics at https://digital.gov/topics
+topics: 
+  - plain-language
+
+# see all authors at https://digital.gov/authors
+authors: 
+  - bryce-spivey
```
Comment (saracope) [inline content/events/2019/06/2019-06-12-how-develop-an-agency-brand-voice-that-connects-with-customers-online.md:25]: I think this is supposed to be katherine-spivey @DanielJPino-ux

Comment (ToniBonittoGSA) [reply content/events/2019/06/2019-06-12-how-develop-an-agency-brand-voice-that-connects-with-customers-online.md]: done

**Comment by ToniBonittoGSA** [comment]
https://federalist-466b7d92-5da1-4208-974f-d61fd4348571.app.cloud.gov/preview/gsa/digitalgov.gov/Event-Page-Update/event/2019/06/12/how-develop-an-agency-brand-voice-that-connects-with-customers-online/

**Pull request merged**